setup.sh

```diff
@@ -3,14 +3,26 @@ if [ ! -d "Dependencies" ]; then
   mkdir Dependencies
 fi
 pushd Dependencies
-if [ ! -d "Catch2-2.2.1" ]; then
-  git clone --branch v2.2.1 https://github.com/catchorg/Catch2.git Catch2-2.2.1
-  if [ -d "Catch2-2.2.1" ]; then
-    pushd Catch2-2.2.1
+if [ ! -d "Catch2-2.6.1" ]; then
+  git clone --branch v2.6.1 https://github.com/catchorg/Catch2.git Catch2-2.6.1
+  if [ -d "Catch2-2.6.1" ]; then
+    pushd Catch2-2.6.1
+    cmake .
+    make
+    make install
+    popd
+  fi
+fi
+python3 -c "from distutils import sysconfig"
```

Comment:
What does line 16 do?

Reply:
Nothing, added there to test for some missing imports but forgot to remove it. Removing.